lldb/source/Plugins/SymbolFile/DWARF/DWARFDIE.cpp

```diff
@@ -50,7 +50,8 @@ class ElaboratingDIEIterator
     m_worklist.pop_back();
 
     // And add back any items that elaborate it.
-    for (dw_attr_t attr : {DW_AT_specification, DW_AT_abstract_origin}) {
+    for (dw_attr_t attr :
+         {DW_AT_specification, DW_AT_abstract_origin, DW_AT_specification}) {
```

Comment:
Did you mean for one of these to be `DW_AT_signature`?

Reply:
Just from cursory reading it seems like `ElaboratingDIEIterator` is only used for method DIEs currently? Which might be why no tests were affected here?

Reply:
Just from cursory reading it seems like `ElaboratingDIEIterator` is only useful for method DIEs currently? Which might be why no tests were affected here?

If you do need it as part of this patch then there's a couple of comments to fix up in `ElaboratingDIEIterator` that specifically talk about `DW_AT_specification`/`DW_AT_abstract_origin`

Reply:
Yes, I meant that to be DW_AT_signature. Oops.

This isn't really needed for this patch, but I thought it makes sense to add it for symmetry. The class is only currently used in `DWARFDIE::IsMethod` which does not make sense for types, but I could imagine this class being used elsewhere as well.